JS-Algorithms/challenges.js

```diff
@@ -41,7 +63,22 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let sizeArray = COUNTRY_NAMES.length;
+  let firstPart = [];
+  let secondPart = [];
+  let array = COUNTRY_NAMES[(index % sizeArray + sizeArray) % sizeArray];
+  let nIndice = COUNTRY_NAMES.indexOf(array)
```

Comment:
please correct me if I'm wrong but it seems this always return the value of the `index` parameter, so why we don't use it directly if that's what you needed. for eg: COUNTRY_NAMES[index]

Reply:
please read a little more about array methods, I think you could have used a single variable to achieve the merge of the two resulting arrays